content/language/python/containerize.md

```diff
@@ -115,10 +174,10 @@ USER appuser
 COPY . .
 
 # Expose the port that the application listens on.
-EXPOSE 8000
+EXPOSE 5000
 
 # Run the application.
-CMD python3 -m flask run --host=0.0.0.0 --port=8000
+CMD python3 -m uvicorn app:app --host=0.0.0.0 --port=5000
```

Comment:
mac users would, by default, already have a service listening on port 5000. Could we maybe choose a different high port, e.g. 8000

Reply:
Good catch, I didn't know there is a service running in Mac OS on 5000. I fix it to 8000. Thanks for the hint